app/models/pawn.rb

```diff
@@ -1,6 +1,29 @@
 class Pawn < Piece
-  def move_valid?
-    # add logic here
+  def move_valid?(x, y)
+    return false if space_occupied?(x, y) == true
+    return false if legal_move?(x, y) == false
```

Comment:
And I'm planning to change move_valid to get rid of space_occupied? because I want to add that instead to one_step_legal so I can allow moving into a space occupied by a diagonally opposing piece to capture it.